testcase/testcase.go

```diff
@@ -5,33 +5,32 @@ package testcase
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/magnusbaeck/logstash-filter-verifier/logging"
+	"github.com/magnusbaeck/logstash-filter-verifier/logstash"
+	"gopkg.in/yaml.v2"
```

Comment:
Isn't the convention to group imports with standard packages at the top and third-party packages below (as in the original code)?

Reply:
Thanks for noticing. :) I am aware of such rules in, say Python. However, I have to admit not having looked up the common style and formatting rules for Go yet. I'll reorder it today.